src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -57,19 +54,10 @@ export class MultisigTransactionInfoMapper {
     const dataSize =
       dataByteLength >= 2 ? Math.floor((dataByteLength - 2) / 2) : 0;
 
-    if (this.isCustomTransaction(value, dataSize, transaction.operation)) {
-      return await this.customTransactionMapper.mapCustomTransaction(
-        transaction,
-        dataSize,
-        chainId,
-      );
-    }
```

Comment:
This is unnecessary as it is the ["default" map of this function](https://github.com/safe-global/safe-client-gateway-nest/blob/refactor-argument/src/routes/transactions/mappers/common/transaction-info.mapper.ts#L114-L118).

Reply:
Not sure about this change because I think order matters here. This is the default but the condition to consider a transaction _custom_ is:
```
(value > 0 && dataSize > 0) || operation !== 0
```
But not sure this overlaps with other conditions which are checked later in the code, so I think we need to check the tests are covering all the cases before doing this change. Maybe it's possible to refactor, but I'd add more exhaustive testing here.

Reply:
Considering the unsurity, I've reverted this. We can revisit it later as increasing the test coverage is a necessity either way.